src/ucp/api/ucp.h

```diff
@@ -104,6 +104,21 @@
 
 /**
  * @ingroup UCP_CONTEXT
+ * @brief UCP context parameters field mask.
+ *
+ * The enumeration allows specifying which fields in @ref ucp_params_t are
+ * present, for backward compatibility.
```

Comment:
The enumeration allows specifying which fields in @ref ucp_params_t are present. It is used for the enablement of backward compatibility support.